src/microservices/users/src/Api/Api.csproj

```diff
@@ -32,4 +32,5 @@
     <ProjectReference Include="..\Infrastructure\Infrastructure.csproj" />
     <ProjectReference Include="..\Persistence\Persistence.csproj" />
   </ItemGroup>
+  <ProjectExtensions><VisualStudio><UserProperties appsettings_1json__JsonSchema="" /></VisualStudio></ProjectExtensions>
```

Comment:
Do we need this?

Reply:
I believe that was somehow generated by Visual Studio. but doesn't look as something we need